### ISSUE TITLE
Frequent virus alert (Potentially unwanted app found) even after EP uninstall

### ISSUE THREAD

**Issue by FLcruising** [valinet/ExplorerPatcher#3030]
Since the update last month that caused Win11 to freak out, I downgraded to 62.2 and also upgraded (pre-release 64.1 now). The stupid virus notification has been popping up ever since many times per day. Uninstalling EP for a week didn't make it go away. How do I stop this annoyance?

**Comment by Amrsatrio** [comment]
If it is reported by Defender, please check the protection history. From there you can see the affected path(s).

**Comment by FLcruising** [comment]
> If it is reported by Defender, please check the protection history. From there you can see the affected path(s).

That's the thing, it's only a popup notification.
![image](https://github.com/valinet/ExplorerPatcher/assets/86490693/0a3f4cc8-cb7c-47da-9a55-e8be71df7d82)
![image](https://github.com/valinet/ExplorerPatcher/assets/86490693/5a864fe5-bff7-455c-b621-3a7810c6b750)

**Comment by Amrsatrio** [comment]
Weird. Nothing as well in "Current threats"? Maybe try clearing the entire data of your web browsers so that remnants of the falsely flagged EP 63.2 and 63.3 are deleted? Try deleting %APPDATA%\ExplorerPatcher as well.

Related Reddit post but no solutions posted:
https://www.reddit.com/r/WindowsHelp/comments/18keijr/windows_defender_showing_potentially_unwanted_app/

**Comment by Amrsatrio** [comment]
Yeah. Gotta make EP delete the entirety of %APPDATA%\ExplorerPatcher on uninstallation. That is for later.
